### DISCUSSION TITLE
Telescope Not Connecting

### DISCUSSION THREAD

**Discussion by VictoryPete** [Stellarium/stellarium#3056] [Q&A]
HI,
  I have a new Primalucelab Eagle 4 now, replacing the Eagle LE I had. Telescope was connecting before I got the new Eagle. My EQ6-R is currently connecting to GSServer and everything connects in NINA. It just doesn't connect in Stellarium now, I have uninstalled and reinstalled Stellarium and still it doesn't connect.

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin

**Reply by VictoryPete** [reply]
Okay, this doesn't really help but thanks. Does GSServer need to be already on and connected for this to connect to Stellarium. I don't remember how I would do it before this happened. NINA opens GSServer by itself.

**Reply by gzotti** [reply]
I don't run a goto telescope currently and cannot answer this question. I want to point out you should use a Qt5-based build, this seems to work better. What we know is in the User Guide.

**Reply by VictoryPete** [reply]
I don't know what a Qt5-based build is.

**Reply by gzotti** [reply]
An installer that has "qt5" in its filename.

**Reply by VictoryPete** [reply]
Not sure what all this means. Just getting the mount to work with GSServer and NINA was a bit of a challenge to begin with.

**Comment by VictoryPete** [comment]
I give up on Stellarium. The Configure button again does nothing. When it did open previously, my mount would not connect.
